[2025-06-20T23:45:15.773+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-20T23:45:15.814+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_dag.dbt_test manual__2025-06-20T23:44:35.340303+00:00 [queued]>
[2025-06-20T23:45:15.835+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_dag.dbt_test manual__2025-06-20T23:44:35.340303+00:00 [queued]>
[2025-06-20T23:45:15.855+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-06-20T23:45:16.037+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): dbt_test> on 2025-06-20 23:44:35.340303+00:00
[2025-06-20T23:45:16.058+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=147) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-20T23:45:16.063+0000] {standard_task_runner.py:72} INFO - Started process 149 to run task
[2025-06-20T23:45:16.062+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'dbt_test', 'manual__2025-06-20T23:44:35.340303+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpaxgkly0h']
[2025-06-20T23:45:16.068+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask dbt_test
[2025-06-20T23:45:16.219+0000] {task_command.py:467} INFO - Running <TaskInstance: dbt_dag.dbt_test manual__2025-06-20T23:44:35.340303+00:00 [running]> on host 488def881868
[2025-06-20T23:45:16.448+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_dag' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2025-06-20T23:44:35.340303+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-20T23:44:35.340303+00:00'
[2025-06-20T23:45:16.450+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-20T23:45:16.498+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-20T23:45:16.501+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/dbt && dbt test --profiles-dir /opt/dbt/.dbt']
[2025-06-20T23:45:16.533+0000] {subprocess.py:99} INFO - Output:
[2025-06-20T23:45:23.152+0000] {subprocess.py:106} INFO - [0m23:45:23  Running with dbt=1.10.2
[2025-06-20T23:45:25.795+0000] {subprocess.py:106} INFO - [0m23:45:25  Registered adapter: snowflake=1.9.4
[2025-06-20T23:45:27.687+0000] {subprocess.py:106} INFO - [0m23:45:27  Found 4 models, 4 data tests, 3 sources, 476 macros
[2025-06-20T23:45:27.696+0000] {subprocess.py:106} INFO - [0m23:45:27
[2025-06-20T23:45:27.698+0000] {subprocess.py:106} INFO - [0m23:45:27  Concurrency: 4 threads (target='dev')
[2025-06-20T23:45:27.700+0000] {subprocess.py:106} INFO - [0m23:45:27
[2025-06-20T23:45:30.565+0000] {subprocess.py:106} INFO - [0m23:45:30  1 of 4 START test not_null_stg_customers_customer_id ........................... [RUN]
[2025-06-20T23:45:30.568+0000] {subprocess.py:106} INFO - [0m23:45:30  2 of 4 START test not_null_stg_sales_product_id ................................ [RUN]
[2025-06-20T23:45:30.571+0000] {subprocess.py:106} INFO - [0m23:45:30  3 of 4 START test not_null_stg_sales_transaction_id ............................ [RUN]
[2025-06-20T23:45:30.578+0000] {subprocess.py:106} INFO - [0m23:45:30  4 of 4 START test unique_stg_customers_customer_id ............................. [RUN]
[2025-06-20T23:45:31.519+0000] {subprocess.py:106} INFO - [0m23:45:31  1 of 4 PASS not_null_stg_customers_customer_id ................................. [[32mPASS[0m in 0.95s]
[2025-06-20T23:45:32.588+0000] {subprocess.py:106} INFO - [0m23:45:32  2 of 4 PASS not_null_stg_sales_product_id ...................................... [[32mPASS[0m in 2.01s]
[2025-06-20T23:45:32.686+0000] {subprocess.py:106} INFO - [0m23:45:32  3 of 4 PASS not_null_stg_sales_transaction_id .................................. [[32mPASS[0m in 2.10s]
[2025-06-20T23:45:32.884+0000] {subprocess.py:106} INFO - [0m23:45:32  4 of 4 PASS unique_stg_customers_customer_id ................................... [[32mPASS[0m in 2.30s]
[2025-06-20T23:45:35.253+0000] {subprocess.py:106} INFO - [0m23:45:35
[2025-06-20T23:45:35.256+0000] {subprocess.py:106} INFO - [0m23:45:35  Finished running 4 data tests in 0 hours 0 minutes and 7.55 seconds (7.55s).
[2025-06-20T23:45:35.655+0000] {subprocess.py:106} INFO - [0m23:45:35
[2025-06-20T23:45:35.658+0000] {subprocess.py:106} INFO - [0m23:45:35  [32mCompleted successfully[0m
[2025-06-20T23:45:35.661+0000] {subprocess.py:106} INFO - [0m23:45:35
[2025-06-20T23:45:35.664+0000] {subprocess.py:106} INFO - [0m23:45:35  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 NO-OP=0 TOTAL=4
[2025-06-20T23:45:37.468+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-20T23:45:37.561+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-20T23:45:37.563+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dbt_dag, task_id=dbt_test, run_id=manual__2025-06-20T23:44:35.340303+00:00, execution_date=20250620T234435, start_date=20250620T234515, end_date=20250620T234537
[2025-06-20T23:45:37.675+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-20T23:45:37.721+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-20T23:45:37.736+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
